### PR TITLE
Bump vodozemac

### PR DESCRIPTION
**Pull request by poljar**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -1264,31 +1274,30 @@ checksum = "487585f4d0c6655fe74905e2504d8ad6908e4db67f744eb140876906c2f3175d"
...
 
 [[package]]
 name = "ed25519"
-version = "1.5.3"
+version = "2.2.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "91cff35c70bba8a626e3185d8cd48cc11b5437e1a5bcd15b9b5fa3c64b6dfee7"
+checksum = "5fb04eee5d9d907f29e80ee6b0e78f7e2c82342c63e3580d8c4f69d9d5aad963"
 dependencies = [
+ "pkcs8",
```
Comment (jplatte) [inline Cargo.lock:1286]: Is this a sensible extra dependency, or maybe something upstream should make optional?

File: Cargo.lock [hunk 2]
```diff
@@ -5470,23 +5504,25 @@ checksum = "49874b5167b65d7193b8aba1567f5c7d93d001cafc34600cee003eda787e483f"
...
+ "curve25519-dalek",
  "ed25519-dalek",
+ "getrandom 0.2.10",
  "hkdf",
  "hmac",
  "matrix-pickle",
  "pkcs7",
  "prost",
- "rand 0.7.3",
+ "rand 0.8.5",
```
Comment (jplatte) [inline Cargo.lock:5521]: So we still depend on old rand after this, ugh. Tracked that down and opened another upstream issue: https://github.com/LukeMathWalker/wiremock-rs/issues/123

**Review by jplatte — APPROVED** [review]

File: Cargo.lock (hunk 1 above)
Comment (poljar) [reply Cargo.lock:1286]: It is already optional, and we don't enable it :thinking: 

https://github.com/dalek-cryptography/curve25519-dalek/blob/e44d4b5903106dde0e5b28a2580061de7dfe8a9f/ed25519-dalek/Cargo.toml#L71

Comment (jplatte) [reply Cargo.lock:1286]: Hm, I can't find out why it would be enabled. Maybe it actually isn't, but Cargo still puts it in the lockfile for some reason.

**Pull request merged**